<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>downloadOptionsDialog</class>
 <widget class="QDialog" name="downloadOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>304</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>443</width>
    <height>304</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>443</width>
    <height>304</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Advanced options for download</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/configure.png</normaloff>:/icons/icons/configure.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="lblThreads">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>10</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Number of parallel connections</string>
   </property>
   <property name="text">
    <string>Use threads</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinThreads">
   <property name="geometry">
    <rect>
     <x>338</x>
     <y>9</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Number of parallel connections</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadOptionDialogOkButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/dialog-ok-apply.png</normaloff>:/icons/icons/dialog-ok-apply.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="lblSocketTimeo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Network timeout in seconds</string>
   </property>
   <property name="text">
    <string>Network Timeout</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinTimeout">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>7</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Network timeout in seconds</string>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="cacheDirLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Cache folder to search for</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cacheDirBrowseButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCacheDir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Cache folder to search for</string>
   </property>
   <property name="text">
    <string>Cache Directory</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="disableChecksumCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Disables checksum verification of downloaded items. Enable only if you know what you are doing</string>
   </property>
   <property name="text">
    <string>Disable Checksum</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="fetchBugReportsCheckBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Fetch Bug Reports</string>
   </property>
   <property name="text">
    <string>Fetch Bug Reports</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="proxyGroupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>169</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLineEdit" name="proxyHostLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>39</y>
      <width>238</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Proxy Server Host/IP Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="proxyPortLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>39</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="toolTip">
     <string>Proxy Server Port Address</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="useProxyCheckBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>89</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Check if using a Proxy server</string>
    </property>
    <property name="text">
     <string>Proxy</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblProxyPort">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>186</y>
     <width>138</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Cache folder to search for</string>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProxyHost">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>186</y>
     <width>138</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Cache folder to search for</string>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinTimeout</tabstop>
  <tabstop>spinThreads</tabstop>
  <tabstop>cacheDirLineEdit</tabstop>
  <tabstop>cacheDirBrowseButton</tabstop>
  <tabstop>disableChecksumCheckBox</tabstop>
  <tabstop>fetchBugReportsCheckBox</tabstop>
  <tabstop>useProxyCheckBox</tabstop>
  <tabstop>proxyHostLineEdit</tabstop>
  <tabstop>proxyPortLineEdit</tabstop>
  <tabstop>downloadOptionDialogOkButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
